packages/firebase_storage/firebase_storage/windows/firebase_storage_plugin.cpp

```diff
@@ -48,6 +48,7 @@ using flutter::EncodableValue;
 
 namespace firebase_storage_windows {
 
+static std::string kLibrarayName = "flutter-fire-gcs";
```

Comment:
```suggestion
static std::string kLibraryName = "flutter-fire-gcs";
```

Again, I think this should be changed to `flutter-fire-storage` to make it clear IMO